packages/orchestrator-ui-components/src/components/WFOTable/WFOBasicTable/WFOBasicTable.tsx

```diff
@@ -8,6 +8,8 @@ import {
     WFOTableControlColumnConfig,
     WFOTableDataColumnConfig,
 } from '../utils/columns';
+import { useOrchestratorTheme } from '../../../hooks';
```

Comment:
More of a generic thing, but I think it's a good idea specify imports relative to a base path, or provide path aliases (should be configurable). so you can write imports as

```
import { MyThing } from '@components/things
import { useOrchestratorTheme } from '@hooks'
```

Reply:
Good one, placed it on our backlog: https://github.com/workfloworchestrator/orchestrator-ui/issues/302